recipe/meta.yaml

```diff
@@ -72,4 +72,4 @@ about:
 extra:
```

Comment:
Could you please remove `downstreams` before merging? Potentially it can cause issues when we'll make build-out for python 3.11

Reply:
Ok